recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.2.1" %}
+{% set version = "1.2.2" %}
 
 package:
   name: scikit-learn
   version: {{ version }}
 
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 50613365b0c118b07ad51902d46b9f7779b8f81ea932a205bd12e64b829eea9a
+  sha256: 992694e21ce4285aab71b09939d3ed7e5ddb41b8803eb98e10aaba927b74bdf1
   patches:
    - patches/0001-ignore-errors-when-tearing-down-modules.patch
```

Comment:
```suggestion
    - patches/0001-ignore-errors-when-tearing-down-modules.patch
```

Reply:
nit to be inline with the proposal of the boat: https://github.com/conda-forge/scikit-learn-feedstock/pull/216/files

Reply:
It's the automatic re-redering that did that. I'm going to merge as is to avoid another 4h CI run. Let's clean that in the next release :)